src/js/serial_backend.js

```diff
@@ -114,8 +115,7 @@ function connectDisconnect() {
         const selected_baud = PortHandler.portPicker.selectedBauds;
         const selectedPort = portName;
 
-        if (selectedPort === 'DFU') {
-            $('select#baud').hide();
+        if (selectedPort === 'noselection' || selectedPort.includes('DFU')) {
```

Comment:
Caution! The port name does not includes DFU in all the cases. The selectedPort, if I'm not wrong, contains the "path" of the device. serial_xxxx for serial and usb_xxxx for dfu. At least you have changed something at other part.

Reply:
Okay will be better to scan for `usb_` then. Will update.

Reply:
Sorry, this is at least after my refactor. If we don't want in this way, we can change it.

Reply:
Yes #3991 should be merged first